docs/introduction-to-smart-contracts.rst

```diff
@@ -119,8 +119,8 @@ that does not allow any arithmetic operations. It is suitable for
 storing addresses of contracts or keypairs belonging to external
 persons. The keyword ``public`` automatically generates a function that
 allows you to access the current value of the state variable.
-Without this keyword, other contracts have no way to access the variable.
-The function will look something like this::
+Without this keyword, other contracts have no way to access the variable,
+ but raw implementation would look something like this::
```

Comment:
I think we should reconsider this example as it uses shadowing and would not compile.

Likely we should just explain what the signature will be.